src/pages/settings/Wallet/ExpensifyCardPage.js

```diff
@@ -74,58 +81,89 @@ function ExpensifyCardPage({
+                                messages={[translate('cardPage.cardLocked')]}
+                                type="error"
+                            />
+                        )}
+
+                        {isDetectedIndividualFraud && !isDetectedDomainFraud && (
                             <>
-                                {shouldShowCardDetails ? (
-                                    <CardDetails
-                                        // This is just a temporary mock, it will be replaced in this issue https://github.com/orgs/Expensify/projects/58?pane=issue&itemId=33286617
-                                        pan="1234123412341234"
-                                        expiration="11/02/2024"
-                                        cvv="321"
-                                        domain={domain}
-                                    />
-                                ) : (
-                                    <MenuItemWithTopDescription
-                                        description={translate('cardPage.virtualCardNumber')}
-                                        title={CardUtils.maskCard(virtualCard.lastFourPAN)}
-                                        interactive={false}
-                                        titleStyle={styles.walletCardNumber}
-                                        shouldShowRightComponent
-                                        rightComponent={
-                                            <Button
-                                                medium
-                                                text={translate('cardPage.cardDetails.revealDetails')}
-                                                onPress={handleRevealDetails}
-                                            />
-                                        }
-                                    />
-                                )}
+                                <DangerCardSection
+                                    title={translate('cardPage.suspiciousBannerTitle')}
+                                    description={translate('cardPage.suspiciousBannerDescription')}
+                                />
                                 <MenuItemWithTopDescription
-                                    title={translate('cardPage.reportFraud')}
+                                    title={translate('cardPage.reviewTransaction')}
                                     titleStyle={styles.walletCardMenuItem}
-                                    icon={Expensicons.Flag}
+                                    icon={Expensicons.MagnifyingGlass}
+                                    iconFill={theme.icon}
                                     shouldShowRightIcon
-                                    onPress={() => Navigation.navigate(ROUTES.SETTINGS_REPORT_FRAUD.getRoute(domain))}
+                                    brickRoadIndicator="error"
+                                    onPress={() => Link.openOldDotLink('inbox')}
```

Comment:
> We should leave a note that this won't work in DEV and staging too, only in production (I wonder if buildOldDotURL is supposed to add environment to URL, since there is no dev/staging version of oldDot)

@0xmiroslav Internal employees do have a DEV/staging Old Dot

Reply:
I guess I'll have to change the profile pic, this isn't the first time this happens 😅